dashboard/test/helpers/delete_accounts_helper_test.rb

```diff
@@ -229,8 +229,51 @@ class DeleteAccountsHelperTest < ActionView::TestCase
     assert user.valid?
   end
 
+  #
+  # Table: dashboard.sections
+  #
+
+  test "soft-deletes all of a user's owned sections" do
+    user = create :teacher
+    create :section, user: user
+    create :section, user: user
+    create :section, user: user
+
+    refute_empty Section.where(user: user)
+    refute_empty Section.with_deleted.where(user: user)
+
+    purge_user user
```

Comment:
It just occurred to me that the deletion of a section as a cascaded effect from deletion of a teacher is going to change when co-teachers land. @davidwufer, something to keep in mind, we'll want to make sure we talk with a PM about exactly how deletion (soft and hard) should work with co-teachers.